bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractFormComponentImpl.java

```diff
@@ -410,7 +458,7 @@ public FormComponentData getData() {
                     this.dataLayerEnabled = com.adobe.cq.wcm.core.components.util.ComponentUtils.isDataLayerEnabled(this.getCurrentPage()
                         .getContentResource());
                 } else {
-                    this.dataLayerEnabled = ComponentUtils.isDataLayerEnabled(this.resource);
+                    this.dataLayerEnabled = com.adobe.cq.wcm.core.components.util.ComponentUtils.isDataLayerEnabled(this.resource);
```

Comment:
why have we removed the import and using full classPath here?

Reply:
as we have started using our own ComponentUtils class.